composer.json

```diff
@@ -37,17 +37,17 @@
     ],
 
     "require": {
-        "php": ">=5.5",
-        "league/climate": "^2.0|^3.0",
+        "php": "^5.6 || ^7.0",
```

Comment:
This is what's listed in the composer documentation for logical OR, but it's got no mention of a single pipe, figure it's worth keeping consistent.